app/frontend/src/components/TextField.tsx

```diff
@@ -21,10 +21,11 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-interface AccessibleTextFieldProps extends BaseTextFieldProps {
+type AccessibleTextFieldProps = Omit<TextFieldProps, "variant"> & {
   id: BaseTextFieldProps["id"];
   onChange?: TextFieldProps["onChange"];
-}
+  variant?: "filled" | "outlined" | "standard";
```

Comment:
I think you can add this and leave it as an interface and this will overrule the `variant` from `TextFieldProps`

Reply:
I can't, I get `An interface can only extend an object type or intersection of object types with statically known members.` when trying to extend `TextFieldProps`